docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+
+services:
+  backend:
+    build:
+      context: ./ninjareact
+    container_name: django_backend
```

Comment:
It's best practice to allow compose to manage container names for services, to avoid namespace conflicts when running multiple instances.